test/controllers/callback_controller_test.rb

```diff
@@ -141,14 +146,25 @@ class CallbackControllerTest < ActionController::TestCase
       end
     end
 
-    test "#callback redirects to the root_url with shop and host parameter" do
+    test "#callback redirects to the root_url with shop and host parameter for non-embedded" do
+      ShopifyApp.configuration.embedded_app = false
+      setup_context
```

Comment:
Same here, the context is set up for tests automatically based on the configs set in `setup`, so we wouldn't need to have a separate method to do that here.